docs/i18n.rst

```diff
@@ -157,13 +157,20 @@ first::
 counter with the name ``count`` and provide a plural translation after the ``{%
 plural %}`` tag::
 
-    {% blocktrans with amount=article.price count years=i.length %}
+    {% blocktrans trimmed with amount=article.price count years=i.length %}
     That will cost $ {{ amount }} per year.
     {% plural %}
     That will cost $ {{ amount }} per {{ years }} years.
     {% endblocktrans %}
 
 
+.. note::
+
+   The previous multi-lines examples also use the ``trimmed`` option.
+   This remove newline characters and replace any whitespace at the beginning and end of a line,
```

Comment:
removes ... replaces

Reply:
Thanks!

3 sentences, 2 mistakes. At least is not 3/3. I'm improving my skills :stuck_out_tongue_closed_eyes:

Reply:
Shouldn’t it be "multi-line examples", (without the s after line), though ?

(Not 100% sure)

Reply:
Uh, yes, it should. 😅

Reply:
OK! I reached the best score: 3/3 :1st_place_medal: 


Thanks!